interface/src/app/status/SystemLog.tsx

```diff
@@ -112,7 +113,7 @@ const SystemLog = () => {
   });
 
   onError(() => {
-    toast.error('No connection to Log server');
+    toast.error('No connection to Log service');
   });
 
   // called on page load to reset pointer and fetch all log entries
```

Comment:
You can remove the fetch request and reset the pointer on getValue, this is also called on page open.

Reply:
not sure I understand. the pointer needs to be reset when the page loads

Reply:
I understand now. Will fix.